tracks/python/exercises/phone-number/mentoring.md

```diff
@@ -0,0 +1,54 @@
+# Mentoring
+
+## Phone Number
+
+The problem asks you to write a `PhoneNumber` class.
+The constructor takes an input string that may or may not contain a valid number.
+The constructor needs to validate the number.
+The `pretty()` method needs to return a formatted number.
+
+## Reasonable Solution
+
+```python
+class Phone:
+    """Phone number validator."""
+
+    def __init__(self, number: str):
+        """Load and clean a number, validating it."""
+        # Ignore all non-digits.
+        digits = ''.join(i for i in number if i.isdigit())
+
+        # For an 11-digit with country code, drop the country code.
+        if len(digits) == 11 and digits.startswith('1'):
+            digits = digits[1:]
+
+        if len(digits) != 10:
+            raise ValueError('Invalid number')
+
+        self.number = digits
+        self.area_code = digits[0:3]
+        self.exchange = digits[3:6]
+        self.subscriber = digits[6:10]
+
+        for invalid in ('0', '1'):
+            for check in (self.area_code, self.exchange):
+                if check.startswith(invalid):
+                    raise ValueError('Invalid number')
```

Comment:
This is another area where getting "invalid number" would be confusing, where an invalid digit would be understandable.  0 and one are truly valid numbers, but invalid digits for this position.

Reply:
The exercise has since been updated to require specific error messages. I will update this solution to match the new requirements.